pkg/kando/location_pull.go

```diff
@@ -62,3 +72,8 @@ func targetWriter(target string) (io.Writer, error) {
 func locationPull(ctx context.Context, p *param.Profile, path string, target io.Writer) error {
 	return location.Read(ctx, target, *p, path)
 }
+
+// kopiaLocationPull pulls the data from a kopia snapshot into the given target
+func kopiaLocationPull(ctx context.Context, backupID, path string, target io.Writer) error {
+	return kopia.Read(ctx, path, backupID, target)
```

Comment:
Looking at `kopia.Read` def, I think we are passing wrong arguments here
```suggestion
	return kopia.Read(ctx, backupID, path, target)
```